packages/blockchain-wallet-v4-frontend/src/modals/Brokerage/Banks/Deposit/WireInstructions/template.success.tsx

```diff
@@ -1,281 +1,31 @@
-import React, { useState } from 'react'
-import { FormattedMessage } from 'react-intl'
+import React from 'react'
 import styled from 'styled-components'
 
-import Currencies from '@core/exchange/currencies'
-import { Icon, Link, TabMenu, TabMenuItem, Text, TextGroup } from 'blockchain-info-components'
-import { DisplayIcon, DisplaySubTitle, DisplayTitle } from 'components/BuySell'
-import { FlyoutWrapper } from 'components/Flyout'
-import { BankDWStepType } from 'data/types'
-
 import { Props as OwnProps, SuccessStateType } from '.'
-import LayoutArs from './Currency/ARS'
-import LayoutDefault from './Currency/default'
-import { TransferType } from './types'
+import LayoutDefault from './CurrencyLayouts/default'
+import LayoutUsdArs from './CurrencyLayouts/LayoutUsdArs'
 
 const Wrapper = styled.div`
   display: flex;
   justify-content: space-between;
   flex-direction: column;
   height: 100%;
 `
-const TopText = styled(Text)`
-  display: flex;
-  justify-content: space-between;
-  flex-direction: row;
-`
-const BackContainer = styled(Text)`
-  display: flex;
-  align-items: center;
-  width: 100%;
-  margin-bottom: 40px;
-  font-weight: 600;
-  font-size: 20px;
-`
-const Bottom = styled(FlyoutWrapper)`
-  display: flex;
-  padding-top: 24px;
-  flex-direction: column;
-  height: 100%;
-`
-const InfoContainer = styled.div`
-  margin-top: 16px;
-`
-const LegalWrapper = styled(TextGroup)`
-  margin-top: 20px;
-`
-const BottomInfoContainer = styled.div`
-  display: flex;
-  flex-direction: column;
-  margin-top: 41px;
-`
-const BottomRow = styled.div`
-  display: flex;
-  flex-direction: row;
-  margin-bottom: 16px;
-`
-const BottomMultiRowContainer = styled.div`
-  position: relative;
-  display: flex;
-  flex-direction: column;
-  color: ${(props) => props.theme.grey800};
-  margin-left: 16px;
-`
-const TabsContainer = styled.div`
-  margin-top: 40px;
-  display: inline-block;
-`
 
 const Success: React.FC<Props> = (props) => {
-  const [transferType, setTransferType] = useState(TransferType.DOMESTIC)
+  const userCountryCode = props.userData?.address?.country || 'default'
+
   const currencyLayouts = {
-    ARS: <LayoutArs {...props} />,
+    ARS: <LayoutUsdArs {...props} />,
+    USD: userCountryCode !== 'AR' ? <LayoutDefault {...props} /> : <LayoutUsdArs {...props} />,
```

Comment:
this doesnt make much sense?  why need this if default is the same as USD? also why checking country code here for the same layout as ARS?